assets/js/libs/ace/ace.js

```diff
@@ -12127,7 +12127,7 @@ top: 0;\
 .ace_scrollbar-h {\
 position: absolute;\
 overflow-x: auto;\
-overflow-y: hidden;\
+overflow-y: auto;\
```

Comment:
we should never patch external dependencies directly. for example it leads to external dependencies that are not upgradable to the next version as we would lose our custom changes we made directly in the libraries.